packages/graphback-codegen-client/tests/snapshots/GraphQLClientCreatorTest.ts.md

```diff
@@ -34,11 +34,7 @@ Generated by [AVA](https://ava.li).
                 ...NoteFields␊
               }␊
             }␊
-          fragment NoteFields on Note {␊
-              id␊
-              title␊
-              description␊
-            }`,
```

Comment:
I've updated the ava snapshots using `ava --update-snapshots` and also all fragments has been removed, is that correct?